packages/etsh/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://etsh.nl
+TERMUX_PKG_DESCRIPTION="An enhanced, backward-compatible port of Thompson Shell"
+TERMUX_PKG_LICENSE="BSD 2-Clause"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=5.4.0
+TERMUX_PKG_SRCURL=https://etsh.nl/src/etsh_5.4.0/etsh-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=fd4351f50acbb34a22306996f33d391369d65a328e3650df75fb3e6ccacc8dce
+TERMUX_BUILD_IN_SRC=true
+
+termux_step_configure() {
+	( cd ${TERMUX_PKG_SRCDIR} ;
+	sh ./mkconfig )
+}
```

Comment:
`cd` is not needed here.

Reply:
Omitting `cd` results in an error, which is odd. `pwd` returns the correct path, but running `sh` or `make` fails.

Reply:
See my comment in https://github.com/termux/termux-packages/pull/14430#discussion_r1065283802.